core/src/main/java/org/mskcc/cbio/portal/util/internal/AccessControlImpl.java

```diff
@@ -84,7 +84,7 @@
 
         //  Then, insert "All" Cancer Types at beginning  //TODO - fix this! It conflicts with ALL (Acute Lymphoid Leukemia)!
         ArrayList<CancerStudy> finalCancerStudiesList = new ArrayList<CancerStudy>();
-        String allCancerStudyTitle = (GlobalProperties.usersMustBeAuthorized()) ?
+        String allCancerStudyTitle = (GlobalProperties.usersMustAuthenticate()) ?
```

Comment:
Oh .. didn't realize we had this method available .. nice find.
I hope this covers the right cases through ... did you check whether this returns false for social_auth ?

Reply:
@sheridancbio yep, it does 
```    public static boolean usersMustAuthenticate()
    {
        // authentication for social_auth is optional
        return (!authenticate.isEmpty() && !authenticate.equals("false") && !authenticate.equals("social_auth"));
    }
```